plugins/multilevel-linear-view/src/MultilevelLinearComparativeView/components/AreaOfInterest.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Typography, useTheme, alpha } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+import { LinearGenomeMultilevelViewModel } from '../../LinearGenomeMultilevelView/model'
+import { MultilevelLinearComparativeViewModel } from '../model'
+
+type LCV = MultilevelLinearComparativeViewModel
+type LGV = LinearGenomeMultilevelViewModel
+
+const useStyles = makeStyles()(theme => ({
+  guide: {
+    pointerEvents: 'none',
+    position: 'absolute',
+    zIndex: 10,
+  },
+}))
+
+const AreaOfInterest = observer(
+  ({
+    model,
+    view,
+    polygonPoints,
+  }: {
+    model: LCV
+    view: LGV
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    polygonPoints: any
+  }) => {
+    const { classes } = useStyles()
+    const { left, right } = polygonPoints
+
+    const theme = useTheme()
+    // @ts-ignore
+    const { tertiary, primary } = theme.palette
+    const polygonColor = tertiary ? tertiary.light : primary.light
+
+    const width = !isNaN(right) && right - left !== 0 ? right - left : 3
```

Comment:
might be able to change to a Math.max(x,3), i think it can become less than 3 making it hard to see